healthcheck.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
-nc -z localhost "$HAPROXY_PORT" || exit 1
+if [ "$BIND_ADDRESS" != "*" ]; then
+	nc -z "$BIND_ADDRESS" "$HAPROXY_PORT" || exit 1
+else
+	nc -z "127.0.0.1" "$HAPROXY_PORT" || exit 1
```

Comment:
probably we still should use `localhost` instead of "127.0.0.1" here?

_To not change the logic that was without a strong need._